homepairs/homepairsUI/src/endpoints/api-request.tsx

```diff
@@ -105,6 +105,44 @@ export const fetchPreferredProviders = (pmId: string) => {
     };
 };
 
+/** 
+* ----------------------------------------------------
+* fetchNetworkProviders
+* ---------------------------------------------------- 
+* Makes a get request to the homepairs backend retrieving all network providera from the account 
+* associatted with the account Email. This function calls the dispatch method and updates the store 
+* upon success.
+* 
+* @param {ServiceProvider} serviceProvider -The object holding in the service provider to be removed
+* @param {string} accountEmail -The email of the associated account. This is used to by the backend to 
+* determine which account needs the specified provider to be removed
+* @param {(error:string) => any} onError -An optional callback function that will handle an error 
+* thrown if the api request fails
+*/
+export const fetchNetworkProviders = (accountEmail: string) => {
```

Comment:
Components must be capitalize. This will break on native devices. This is not a problem on web.

Reply:
This was the wrong file